src/coreclr/tools/Common/TypeSystem/Common/MetadataFieldLayoutAlgorithm.cs

```diff
@@ -304,17 +304,17 @@ protected virtual void FinalizeRuntimeSpecificStaticFieldLayout(TypeSystemContex
         {
         }
 
-        protected static ComputedInstanceFieldLayout ComputeExplicitFieldLayout(MetadataType type, int numInstanceFields)
+        protected ComputedInstanceFieldLayout ComputeExplicitFieldLayout(MetadataType type, int numInstanceFields)
```

Comment:
Is there a reason why you changed this from static to non-static?

Reply:
At one point it was needed because there used to be a helper method calculating the offset bias. I have double checked it's not longer necessary after all the refactorings, will remove, thanks for pointing that out!